AnimatedComponents/src/components/floating-button/FloatingButtonDemo.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { Alert } from 'react-native'
+
+import { FileIcon, FolderIcon, PenIcon } from '@assets'
+import { FloatingButtonPreset } from '@constants'
+
+import FloatingButton from './FloatingButton'
+
+const FloatingButtonDemo = () => {
+  const icons = [
+    {
+      icon: FileIcon,
+      onPress: () => Alert.alert('File Icon Pressed'),
+      iconName: 'File',
+    },
+    {
+      icon: FolderIcon,
+      onPress: () => Alert.alert('Folder Icon Pressed'),
+      iconName: 'Folder',
+    },
+    {
+      icon: PenIcon,
+      onPress: () => Alert.alert('Pen Icon Pressed'),
+      iconName: 'Pen',
+    },
+    {
+      icon: PenIcon,
+      onPress: () => Alert.alert('Pen Icon Pressed'),
+      iconName: 'Pencil',
+    },
+  ]
+
+  return <FloatingButton buttonType={FloatingButtonPreset.CircularFloatingButton} icons={icons} />
```

Comment:
can we use different color for heading, red seems too much vibrant? also where from this heading is shown don't see it in demo file?
also can we not export all the variants from here?

Reply:
This is done only for GIF purpose and we cannot give heading to floating button as we did for other components because it is fixed.
If we export all the variants, then it will be very clumsy and we are showing only one variant at a time in every component. I choose red colour because in other PRs as well red was choosen in the GIFs.

Reply:
okay fine for now we will see this when all PR merged.